charts/astronomer/values.yaml

```diff
@@ -125,6 +129,10 @@ houston:
 
   podAnnotations: {}
 
+  podSecurityContext:
+    fsGroup: 1000
+    runAsGroup: 1000
+    runAsUser: 1000
```

Comment:
why are explicitly setting these values now.

Reply:
same for houston as well we didnt had this values before - if we add this we have to handle them for openshift as well